src/config/config.go

```diff
@@ -5,11 +5,13 @@ import "github.com/newrelic/infra-integrations-sdk/args"
 type ArgumentList struct {
 	args.DefaultArgumentList
 	HostRoot              string `default:"" help:"If the integration is running from a container, the mounted folder pointing to the host root folder"`
+	Ecs                   bool   `default:"false" help:"If true and if cgoupsv2 are detected, metrics are collected through Docker API automatically changing the UseDockerAPI default to true"`
```

Comment:
```suggestion
	Ecs                   bool   `default:"false" help:"If true and metrics are collected through the Docker API when cgroupsv2 are detected, it has no effect otherwise"`
```